bitnami/mariadb-galera/values.yaml

```diff
@@ -158,14 +158,14 @@ ldap:
   ## Enable LDAP support
   ##
   enabled: false
-  # uri:
-  # base:
-  # binddn:
-  # bindpw:
-  # bslookup:
-  # nss_initgroups_ignoreusers: root,nslcd
-  # scope:
-  # tls_reqcert:
+  uri:
+  base:
+  binddn:
+  bindpw:
+  bslookup:
+  nss_initgroups_ignoreusers: root,nslcd
+  scope:
+  tls_reqcert:
```

Comment:
What values are mandatory? I'd add a `mariadb-galera.validateValues` function (see other charts such as MinIO) where you can validate the values. 

For instance, if someone sets `ldap.enabled=true` and doesn't provide the `ldap.uri` we could show an error explaining the expected values when enabling LDAP.